recipes-extended/xen/files/libxl-RFC-fix-multiple-disks.patch

```diff
@@ -41,7 +41,22 @@ PATCHES
                  target_path = "/dev/xvdc";
              } else {
                  if (format == NULL) {
-@@ -1605,8 +1605,10 @@ static int libxl__build_device_model_arg
+@@ -1592,6 +1592,14 @@ static int libxl__build_device_model_arg
+                              "qemu-xen doesn't support read-only AHCI disk drivers");
+                         return ERROR_INVAL;
+                     }
++                    if (b_info->stubdomain_version == LIBXL_STUBDOMAIN_VERSION_LINUX) {
++                        target_path = (char *[]) {"/dev/xvda",
```

Comment:
This is a 6 disk duplication of the IDE case below.  Maybe we should use something like the proposed upstream linux-stubdom support:
https://github.com/marmarek/xen/commit/060e89a5cca415871c19d05853a31e7d7bc7f6cc#diff-b43617695b9412434577d745e4f3102dR1820

That's a single setting of target_path for all disks.  That allows 26 disks, but you must need emulated scsi to use more than 6 of them.

Another option would be to move your 6 disk code earlier and let it apply to both AHCI or IDE.

Reply:
I thought of this, but I didn't wanted to mix IDE and AHCI code path and trigger any unknown issues in IDE path.

Reply:
Okay.